modules/caas/backend/src/main/java/io/cattle/platform/inator/util/InstanceFactory.java

```diff
@@ -64,20 +63,9 @@ protected static void addDns(Map<String, Object> fields, StackWrapper stack, Rev
             return;
         }
         List<Object> dns = new ArrayList<>(CollectionUtils.toList(fields.get(InstanceConstants.FIELD_DNS_SEARCH)));
```

Comment:
Is this code really doing anything now?  I just gets and sets the field.  It will have the side effect of always having a non-null value